subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -406,21 +406,34 @@ void ll_reset(void)
 
 	/* Reset LLL via mayfly */
 	{
-		u32_t retval;
-		struct k_sem sem;
 		static memq_link_t link;
 		static struct mayfly mfy = {0, 0, &link, NULL,
 					    perform_lll_reset};
+		u32_t retval;
+
+		/* NOTE: If Zero Latency Interrupt is used then LLL context
+		 *       will be higher priority hence completing inline
```

Comment:
higher priority than? Please clarify in the note :)

Reply:
Updated the note.